docker/.env

```
@@ -0,0 +1,51 @@
+# ROS settings
+ROS_DOMAIN_ID=0
+USE_SIM_TIME=true
```

Planning Simulator doesn't publish /clock topic so this should be false.
(we should set it true when we do work with logging simulator)
```suggestion
USE_SIM_TIME=false
```